pkg/controller/tektoninstallation/tektoninstallation_controller_test.go

```diff
@@ -60,10 +84,122 @@ func TestTektonInstallationController(t *testing.T) {
 				Exists().
 				HasSpec(tektonSub.Spec)
 
+			AssertThatTektonInstallation(t, tektonInstallation.Namespace, tektonInstallation.Name, cl).
+				HasConditions(Unknown())
+		})
+
+	})
+
+	// reconciling on tektonconfig resource watcher
+	t.Run("tektonconfig watcher", func(t *testing.T) {
+
+		t.Run("installed tekton installation", func(t *testing.T) {
+			// given
+			tektonInstallation := NewInstallation()
+			installedCode := []config.ConfigCondition{
+				config.ConfigCondition{
+					Code: "applied-addons",
+				},
+				config.ConfigCondition{
+					Code: config.InstalledStatus,
+				},
+				config.ConfigCondition{
+					Code: "validated-pipeline",
+				},
```

Comment:
this set of config conditions look very similar to the one in previous test - what is the difference?

Reply:
@MatousJobanek, the previous test was to ensure that if we pick up a status that isn't one of the recogonized ones we would set the status to unknown.. in this case, there is a installed status..

Reply:
but the list of the conditions are the same
https://github.com/codeready-toolchain/toolchain-operator/blob/7f06ceaaedf9f5659a3d448b3cbd2c0b132127ad/pkg/controller/tektoninstallation/tektoninstallation_controller_test.go#L58-L109
both of them contain:
```go
			installedCode := []config.ConfigCondition{
				config.ConfigCondition{
					Code: "applied-addons",
				},
				config.ConfigCondition{
					Code: config.InstalledStatus,
				},
				config.ConfigCondition{
					Code: "validated-pipeline",
				},
```
correct me if I'm wrong

Reply:
@MatousJobanek i believe this was changed on the latest changes - please have a look. Also the tests above were old tests which i just updated with config values to make them work. These were tests that i wrote... so the purpose of the tests i guess are different. If you'd like me to remove a test, let me know and i can do that.

Reply:
it looks like you fixed it as part of this commit https://github.com/codeready-toolchain/toolchain-operator/pull/67/commits/693e18e3457ac96cdf66c844fe75b8f71111c215

> Also the tests above were old tests which i just updated with config values to make them work. These were tests that i wrote... so the purpose of the tests i guess are different.

Don't forget that it doesn't really matter if the tests already existed or you created them - they verify the controller logic that was changed by you in your PR, so you should check all already existing tests if they are still valid and alternatively modify them. 

> If you'd like me to remove a test, let me know and i can do that.

there is no point in removing a test without understanding its purpose ;-)

But as I already said - it's fixed, so it's fine :-)